core/payments/reservation/reservation_ledger_config.go

```diff
@@ -0,0 +1,64 @@
+package reservation
+
+import (
+	"fmt"
+	"time"
+)
+
+// Configuration for a ReservationLedger, which manages the reservation of a single account
+type ReservationLedgerConfig struct {
+	// Contains the parameters of the reservation that the ReservationLedger is responsible for
+	reservation Reservation
```

Comment:
```suggestion
	reservation *Reservation
```

Reply:
Let's have the conversation here https://github.com/Layr-Labs/eigenda/pull/1873#discussion_r2277526418